unitex/build.xml

```diff
@@ -63,7 +63,9 @@
 
 	<target name="dist" depends="clean,compile" description="generate the distribution">
 		<!-- create Unitex.jar -->
-		<jar jarfile="${dist-dir}/Unitex.jar" manifest="classes/fr/umlv/unitex/Manifest.mf" basedir="classes" excludes="classes/fr/loria/**" />
+		<jar jarfile="${dist-dir}/Unitex.jar" manifest="classes/fr/umlv/unitex/Manifest.mf" basedir="classes" excludes="classes/fr/loria/**">
+			<zipgroupfileset dir="${lib-dir}" includes="juniversalchardet-1.0.3.jar"></zipgroupfileset>	
```

Comment:
Do not include third-party dependencies without checking before if they are distributed under the LGPLv2.1 or under a more permissive and compatible license; if so, add a copy of the license on gramlab-ide/licenses and update the [LICENSE](https://github.com/UnitexGramLab/gramlab-ide/blob/master/LICENSE) file